src/main/java/com/google/enterprise/cloudsearch/fs/FsRepository.java

```diff
@@ -1519,4 +1579,8 @@ public void close() {
   static interface RepositoryEventPusher {
     ListenableFuture<List<GenericJson>> push(ApiOperation event);
   }
+
+  static interface AllowFilter {
```

Comment:
Add @FunctionalInterface (I don't think we're at all consistent, but it's recommended).